testing/it/e2e-selenium-utils/pom.xml

```diff
@@ -0,0 +1,107 @@
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- ====================================================================== -->
+    <!-- P A R E N T  P R O J E C T  D E S C R I P T I O N                      -->
+    <!-- ====================================================================== -->
+    <parent>
+        <groupId>com.adobe.cq</groupId>
+        <artifactId>core.wcm.components.parent</artifactId>
+        <version>2.17.11-SNAPSHOT</version>
+        <relativePath>../../../parent/pom.xml</relativePath>
+    </parent>
+
+
+    <!-- ====================================================================== -->
+    <!-- P R O J E C T   D E S C R I P T I O N                                  -->
+    <!-- ====================================================================== -->
+    <groupId>com.adobe.cq</groupId>
+    <artifactId>core.wcm.components.it.e2e-selenium-utils</artifactId>
+    <version>2.17.11-SNAPSHOT</version>
+    <name>Adobe Experience Manager Core WCM Components and Utils</name>
+    <packaging>jar</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <configuration>
+                    <excludes combine.children="append">
+                        <!-- exclude comparison files -->
+                        <exclude>**/resources/*.txt</exclude>
+                        <exclude>**/*.md</exclude>
+                        <exclude>build/</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <!-- This is the release profile. -->
+            <id>release</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-jar-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>empty-javadoc-jar</id>
```

Comment:
Since this is a proper jar file, it should have proper javadoc and sources (as required by Maven Central: https://central.sonatype.org/publish/requirements/)

Reply:
Went through the docs and seems GPG siging will also be required. 
There will be a need of GPG key pair for this.
@vladbailescu  Do you know of procedure for adobe specific jars?

Reply:
@vladbailescu  Made required changes. GPG key is configure at parent pom level. So not configuring at artifact level as release will be done using the parent pom.